contessa/rules.py

```diff
@@ -80,8 +80,8 @@ def apply(self, conn: Connector):
 class OneColumnRuleSQL(SqlRule):
     executor_cls = SqlExecutor
 
-    def __init__(self, name, column, **kwargs):
-        super().__init__(name, **kwargs)
+    def __init__(self, name, column, description = None, **kwargs):
```

Comment:
maybe I would not have default here, so that we're sure that each `OneColumnRule` has the description, what do you think ?